### PR TITLE
Add intelmap -> Google Map translation

### PR DESCRIPTION
**Pull request by davidgiesberg**
Hubot will listen for someone to post an intel map link, and respond with a Google Maps static image of the location and a Google Maps link.

```
Someone> https://www.ingress.com/intel?ll=30.318951,-97.7948&z=10
Hubot> https://maps.googleapis.com/maps/api/staticmap?center=30.318951,-97.7948&size=400x300&zoom=15&markers=30.318951,-97.7948
Hubot> https://maps.google.com/maps?ll=30.318951,-97.7948&q=30.318951,-97.7948
```

### CODE REVIEW

File: src/intel-map.coffee [hunk 1]
```diff
@@ -41,6 +42,25 @@ module.exports = (robot) ->
...
+
+  sendGmapLink = (msg, result) ->
+    url = gmapLink result
+    return msg.send url
+
   robot.respond /(intelmap|intel map)(?: for)?\s(.*)/i, (msg) ->
     location = msg.match[2]
     lookupLatLong msg, location, sendIntelLink
+
+  robot.hear /https:\/\/www.ingress.com\/intel\?ll=(-?\d+\.\d+,-?\d+\.\d+)/, (msg) ->
```
Comment (therealklanni) [inline src/intel-map.coffee]: Let's just remove the `https:\/\/www.` from this regex, so it can be more flexible.

**Comment by therealklanni** [comment]
This seems like a good use case for the `slack-attachment` event. Here's an [example](https://github.com/hubot-scripts/hubot-slack-invite-request/blob/master/src/join-request.coffee#L92-L117). This will make the response look better plus reduce the number of messages (most teams are on the free plan so are limited to 10k message history, no sense in sending more messages than we need to from a bot).

**Comment by therealklanni** [comment]
It would also be a good idea to put this feature behind a config option (opt-in would be best, I think), because not every team will necessarily want every Intel Map URL to trigger a bot response. Generally a good idea to put messages that could be potentially seen as "unsolicited" behind a config option, where normally other bot interactions require addressing the bot by name, so a response is expected.

**Comment by drstevens** [comment]
I've found people to be very sensitive to bot _spam_, especially messages from bots which are not explicitly asked for. Our bot is named "sambot" (long story) and has been nicknamed "spambot". At a minimum, this should be configurable.

**Comment by therealklanni** [comment]
Please reopen this issue/pull-request over at [therealklanni/hubot-ingress](https://github.com/therealklanni/hubot-ingress).

**Pull request closed without merging**